inst/htmlwidgets/jexcel.js

```diff
@@ -9,38 +9,48 @@ HTMLWidgets.widget({
 
     return {
       renderValue: function(params) {
-        const { rowHeight,style, ...otherParams } = params;
+        var rowHeight = params.hasOwnProperty("rowHeight") ? params.rowHeight : undefined;
+        var style = params.hasOwnProperty("style") ? params.style : undefined;
+        var otherParams = {};
+        Object.keys(params).forEach(function(ky) {
+          if(params !== "rowHeight" && params !== "otherParams") {
+            otherParams[ky] = params[ky];
+          }
+        });
 
-        const rows = (() => {
+        var rows = (function() {
           if (rowHeight) {
             const rows = {};
             rowHeight.map(data => (rows[data[0]] = { height: `${data[1]}px` }));
             return rows;
           }
           return {};
         })();
-        
-        const formattedStyle = (()=>{
+
+        // this is the only one I do not think I got right
+        // also on the js docs I do not see an initialization property called formattedStyle
+        // https://bossanova.uk/jexcel/v2/docs/quick-reference
+        var formattedStyle = (function() {
```

Comment:
Table styling is mentioned (here)[https://bossanova.uk/jexcel/v3/examples/table-style].

You could do something like this:
``` 
         var formattedStyle = (function() {
          if(style){
            return Object.keys(style).reduce(function(acc, cur) {
              return Object.assign({}, acc, {[cur]:style[cur].join(';')});
            },{});
          } else {
            return {};
          }
        })();
```

Reply:
Got it.  I see now what you are trying to achieve.  Believe it or not, I think even `Object.assign` was not available in the older versions.  I'll rewrite to remove `Object.assign`.  Thanks!!!!

Reply:
I think `Object.assign` was available in es5.

Reply:
Actually, unless I am misunderstanding the anticipated `style` argument in `R`, I don't see why we need this at all.

I'm thinking 

```
excelTable(
  data = matrix(1:100, ncol = 10),
  style = list(
    "A1" = 'background-color: orange;',
    "B1" = 'background-color: orange;',
    "C1" = 'background-color: orange;',
    "D1" = 'background-color: orange;'
  )
)
```

In this case, the `style` can be just passed directly to `jExcel`.

Reply:
Yes you are right. If we pass the argument as you have mentioned we can pass it directly jExcel. Also, the way you suggested makes it simpler in R too.

Reply:
Ok, thanks. I'll amend the pull.

Reply:
https://github.com/timelyportfolio/excelR/commit/f75a22ac2b1fba31111fd99fd954b570ca4f9908

Reply:
Let's change the `Roxygen` comments in [these lines](https://github.com/Swechhya/excelR/blob/7b7c410afc29422800d0757460af61d0cbd7b137/R/jexcel.R#L52-L53) as per the new format.

Reply:
does https://github.com/timelyportfolio/excelR/commit/341e193fcbc0b5e1432522b585f34a005b5935df sound ok?

Reply:
Sounds good! Will go ahead and merge the pull request